lms/templates/course_modes/choose.html

```diff
@@ -72,17 +323,39 @@ <h3 class="title">${_("Sorry, there was an error when trying to enroll you")}</h
             <div class="wrapper-register-choose wrapper-content-main">
                 <article class="register-choose content-main">
                     <header class="page-header content-main">
-                        <h3 class="title">
+                        <h3 class="title v1">
                             ${title_content}
                         </h3>
+                        <h3 class="title v2 hidden">
+                            Next, Select Your Learning Path
+                        </h3>
                     </header>
 
                     <form class="form-register-choose" method="post" name="enrollment_mode_form" id="enrollment_mode_form">
                         <%
                         b_tag_kwargs = {'b_start': HTML('<b>'), 'b_end': HTML('</b>')}
                         %>
                         % if "verified" in modes:
-                            <div class="register-choice register-choice-certificate">
+                            <div class="register-choice register-choice-certificate v2 hidden">
+                                <h4 class="title">Pursue a Verified Certificate</h4>
+                                <div class="wrapper-copy-inline">
+                                  <div class="wrapper-copy-inline">
+                                    <div class="wrapper-copy">
+                                        Get premium level support throughout the course and commit to completion up front.
+                                    </div>
+                                        <img src="https://www.edx.org/sites/all/themes/atedx/images/cert-verified-thumb.png" class="visual-reference img-certificate" alt="Visual representation of a Certificate"/>
```

Comment:
I'd include a note with a ticket number indicating that this is meant to be temporary, at the very least to excuse the hardcoded URLs in here. You may want to consider committing these image and linking to those local version here instead of hardcoding link to our marketing site.